src/hobbits-gui/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ file(GLOB RCFILES "${CMAKE_CURRENT_SOURCE_DIR}/*.qrc" "${CMAKE_CURRENT_SOURCE_DI
 
 
 if (BUILDING_WITH_CONAN)
-	set(ALL_LINK_LIBS hobbits-python hobbits-widgets hobbits-core CONAN_PKG::qt CONAN_PKG::fftw)
+	set(ALL_LINK_LIBS hobbits-python hobbits-widgets hobbits-core CONAN_PKG::qt CONAN_PKG::pffft)
```

Comment:
I think that `hobbits-gui` and `hobbits-runner` actually no longer need to link to the fft library because they don't need to do that "make planner thread safe" call.